miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/service/statsdb/PerPositionBaseContent.java

```diff
@@ -0,0 +1,26 @@
+package uk.ac.bbsrc.tgac.miso.webapp.service.statsdb;
+
+import java.sql.SQLException;
+import java.util.Map;
+
+import uk.ac.tgac.statsdb.run.ReportTable;
+import uk.ac.tgac.statsdb.run.ReportsDecorator;
+import uk.ac.tgac.statsdb.run.RunProperty;
+
+public class PerPositionBaseContent extends TransformReport {
+  public static final PerPositionBaseContent INSTANCE = new PerPositionBaseContent();
+  public PerPositionBaseContent() {
+    super("statsdb-per-position-base-content");
+    add("base_content_a", "base", 0);
```

Comment:
This first argument is the same as the line below, any chance that'll cause issues?

Reply:
No, this is duplicating content.